carbonmark/components/Footer/index.tsx

```diff
@@ -22,54 +21,40 @@ interface Props {
 export const Footer: FC<Props> = (props) => {
   return (
     <footer className={cx(styles.footer(props.transparent), props.className)}>
-      <div className={cx(styles.footer_content, "footer_content")}>
-        <nav className={cx(styles.footer_nav, "footer_nav")}>
-          <Link href="/blog/privacy-policy">
-            <Text t="body4">
-              <Trans>Privacy Policy</Trans>
-            </Text>
-          </Link>
-          <Link href="/blog/terms-of-use">
-            <Text t="body4">
-              <Trans>Terms of Use</Trans>
-            </Text>
-          </Link>
-          <Link href={carbonmarkUrls.docs}>
-            <Text t="body4">
-              <Trans>Help</Trans>
-            </Text>
-          </Link>
-          <Link href="/resources">
-            <Text t="body4">
-              <Trans>Resources</Trans>
-            </Text>
-          </Link>
-          <Link href={urls.home}>
-            <Text t="body4">
-              <Trans>KlimaDAO</Trans>
-            </Text>
-          </Link>
-          <Link href={urls.carbonmarkContactForm}>
-            <Text t="body4">
-              <Trans>Contact</Trans>
-            </Text>
-          </Link>
-        </nav>
-        <nav className={styles.footer_icons}>
-          <A href={urls.twitterCarbonmark}>
-            <TwitterIcon />
-          </A>
-          <A href={urls.github}>
-            <GithubIcon />
-          </A>
-          <A href={urls.linkedInCarbonmark}>
-            <LinkedInIcon />
-          </A>
-          <A href={urls.carbonmarkEmail}>
-            <EmailRoundedIcon />
-          </A>
-        </nav>
-      </div>
+      <nav className={cx(styles.footerNav)}>
```

Comment:
Nit: cx doesn't look to be needed here because there is no additional classes being appended.

```suggestion
      <nav className={styles.footerNav}>
```